Makefile

```diff
@@ -26,13 +27,18 @@ ubuntu-dev-setup:
 	sudo apt-get update && sudo apt-get install -y \
 	  build-essential gcc librados-dev librbd-dev
 
-build:osdsdock osdslet osdsapiserver osdsctl
+build:osdsdock osdslet osdsapiserver osdsctl osds_verify osds_unit_test osds_integration_test osds_e2eflowtest_build osds_e2etest_build 
 
 prebuild:
 	mkdir -p $(BUILD_DIR)
 
 .PHONY: osdsdock osdslet osdsapiserver osdsctl docker test protoc
 
+
```

Comment:
Is there any reason why these blank lines been added? If not, please remove them.